Sources/muterCore/TestReporting/PlainText/testReportTableGeneration.swift

```diff
@@ -52,9 +52,13 @@ func generateMutationScoresCLITable(from fileReports: [MuterTestReport.FileRepor
 // MARK: - Coloring Functions
 func applyMutationTestResultsColor(to rows: [CLITable.Row]) -> [CLITable.Row] {
     return rows.map {
-        let coloredValue = $0.value == TestSuiteOutcome.failed.asMutationTestOutcome ?
-            $0.value.green :
-            $0.value.red
+        let coloredValue = [
+            TestSuiteOutcome.passed.asMutationTestOutcome,
+            TestSuiteOutcome.buildError.asMutationTestOutcome
+        ].contains($0.value) ?
+            $0.value.red :
+            $0.value.green
```

Comment:
I prefer this because of symmetry and readability. Edited on GitHub, so confirm indentation in Xcode if you go with this.
```suggestion
        ].contains($0.value)
            ? $0.value.red
            : $0.value.green
```